### PR TITLE
HDDS-9255. LeaseRecovery CLI Should work for O3fs path.

### PR DESCRIPTION
**Pull request by LZD-PratyushBhatt**
## What changes were proposed in this pull request?

Currently, the Lease Recovery CLI only checks for RootedOzoneFileSystem, so any o3fs path throws an unsupported filesystem error. The aim is to make the CLI work for OzoneFileSystem too.

## What is the link to the Apache JIRA

[HDDS-9255](https://issues.apache.org/jira/browse/HDDS-9255)

## How was this patch tested?
Added a Robot test for the change.

### CODE REVIEW

**Review by jojochuang — COMMENTED** [review]
Other than that it looks good.

There's also a TestLeaseRecoverer but it's a good idea to have robot test coverage as well.
Do you think we should add the same for ofs?

**Comment by LZD-PratyushBhatt** [comment]
> Other than that it looks good.
> 
> There's also a TestLeaseRecoverer but it's a good idea to have robot test coverage as well. Do you think we should add the same for ofs?

Sure, I'll add the same for ofs too.

**Comment by LZD-PratyushBhatt** [comment]
> Other than that it looks good.
> 
> There's also a TestLeaseRecoverer but it's a good idea to have robot test coverage as well. Do you think we should add the same for ofs?

Sure, I'll add the same for ofs too.

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/LeaseRecoverer.java [hunk 1]
```diff
@@ -74,6 +75,8 @@ public Void call() throws Exception {
     FileSystem fs = FileSystem.get(uri, configuration);
     if (fs instanceof RootedOzoneFileSystem) {
       ((RootedOzoneFileSystem) fs).recoverLease(new Path(uri));
+    } else if (fs instanceof OzoneFileSystem) {
+      ((OzoneFileSystem) fs).recoverLease(new Path(uri));
```
[the remark this answers is not in the document]
Comment (LZD-PratyushBhatt) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/LeaseRecoverer.java]: Made the changes, please review. Thanks!

**Review by jojochuang — APPROVED** [review]

**Pull request merged**